iiod/usb-dmabuf.c

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: LGPL-2.1-or-later
+/*
+ * libiio - Library for interfacing industrial I/O (IIO) devices
+ *
+ * Copyright (C) 2022 Analog Devices, Inc.
```

Comment:
2023? not sure if what matters is the time when the file was created or when it's merged into the project...